docs/release-notes/8.8.asciidoc

```diff
@@ -9,6 +9,7 @@
 [[known-issue-8.8.2]]
 ==== Known issues
 
+* Rule changes can't be saved if the rule's action frequency is shorter than the rule's run interval.
```

Comment:
@e40pud thanks for double-checking this. How does look?

```suggestion
* Rule changes can't be saved and existing rule actions are removed if the action's frequency is shorter than the rule's run interval.
```

Reply:
Perfect! Thank you!